gamestonk_terminal/screener/presets/README.md

```diff
@@ -2,6 +2,18 @@
 
 * [template](#template)
 * [sexy_year](#sexy_year)
+* [buffett_like](#traceabl3)
```

Comment:
Here should be: 
[buffett_like](#buffett_like)
[cheap_bottom_dividend](#cheap_bottom_dividend)

and so on!

Reply:
no brackets? or all in one set of brackets?

Reply:
Sorry, it formatted as markdown lol.

I meant like this:
```
[buffett_like](#buffett_like)
[cheap_bottom_dividend](#cheap_bottom_dividend)
```

Reply:
ah... got it.

Reply:
Done.